.github/workflows/post-apiview.yml

```
@@ -0,0 +1,29 @@
+name: After APIView
+
+on:
+  check_run:
+    types: [completed]
+    
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  event-handler:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: ${{ github.event.check_run.check_suite.app.name == 'Azure Pipelines' && 
+          ( contains(github.event.check_run.name, 'APIView') ||
+            contains(github.event.check_run.name, '(SDK Generation Job)') ||
```

I intended to match part of the name for 'SDK Generation' Job. The reason is currently we have 'Preview' in the name and it will be removed when they fully replace the old SDK checks.
```suggestion
            contains(github.event.check_run.name, '(SDK Generation') ||
```